static/app/components/events/eventReplay/staticReplayPreview.tsx

```diff
@@ -66,7 +66,7 @@ export function StaticReplayPreview({
     >
       <PlayerContainer data-test-id="player-container">
         {replay?.hasProcessingErrors() ? (
-          <ReplayProcessingError processingErrors={replay.processingErrors()} />
```

Comment:
is `replay.processingErrors()` used anywhere anymore? might be possible to remove it as well